src/main/resources/test_files/runtime-spark/RumbleML/EstimatorTests/MLEstimator-LogisticRegression1.jq

```diff
@@ -1,14 +1,20 @@
-(:JIQS: ShouldRun; Output="({ "label" : 0, "prediction" : 0 }, { "label" : 1, "prediction" : 1 }, { "label" : 1, "prediction" : 1 }, { "label" : 1, "prediction" : 1 }, { "label" : 1, "prediction" : 1 })" :)
+(:JIQS: ShouldRun; Output="({ "label" : 0, "prediction" : 0 }, { "label" : 1, "prediction" : 1 }, { "label" : 2, "prediction" : 2 }, { "label" : 3, "prediction" : 3 }, { "label" : 4, "prediction" : 4 }, { "label" : 5, "prediction" : 5 })" :)
+let $data := annotate(
+    json-file("./src/main/resources/queries/rumbleML/sample-ml-data-flat.json"),
+    { "label": "integer", "name": "string", "age": "double", "weight": "double", "sentence": "string", "null": "null", "array": ["double"] }
+)
+
 let $est := get-estimator("LogisticRegression")
 let $tra := $est(
```

Comment:
Side comment: best practice is to use complete names for variable names, for readability.

Reply:
Of course!  will try to improve these